src/renderer/hooks/useMutationObserver.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
```

Comment:
// Observe changes in DOM
// API docs: https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver